src/features/allShop/FilterProduct.tsx

```diff
@@ -64,26 +70,68 @@ const FilterButton = styled.button`
   margin: 0px 25px 25px 0;
 `;
 
-export default function FilterProduct() {
+interface FilterProductProps {
+  categories: Category[];
+  onFilterChange: (filterParams: any) => void;
+}
+
+enum FilterProductEnum {
+  ALL = 0,
```

Comment:
```suggestion
  ALL,
```

굳이 숫자 붙이지 않아도 아무것도 안붙이면 자동으로 number로 값이 매핑됩니다

Reply:
![CleanShot 2025-02-13 at 20 58 18@2x](https://github.com/user-attachments/assets/2a9e5e17-2ae6-43d1-b74e-5bc6c841b224)
네 이런식으로 0부터 매핑이 됩니다